app/controllers/feedback_controller.rb

```diff
@@ -31,6 +32,6 @@ def verify_access
   def feedback_params
     params.require(:feedback)
           .permit(:feedback, :contact_email)
-          .merge(application: session[:redirect], username: current_user.display_name)
+          .merge(application: app_display_name, username: current_user.display_name)
```

Comment:
Instead of ...? We have redirect for the link, and we have no app parameter. We retrieve app name based on redirect parameter. In the code above, it is helper method called.